ecleankernel/__main__.py

```diff
@@ -61,6 +63,9 @@ def friendly_desc(self) -> str:
 as root, or preferably mount /boot before using it.'''
 
 
+HOOK_DIR = Path("/etc/eclean-kernel/hook.d")
```

Comment:
EK uses XDG config dirs, see `__main__.py`.

Reply:
Aren't XDG config directories exclusively per user? This is supposed to be a system-wide hook, since `eclean-kernel` is invoked as part of the system management, hence it placed the hooks under `/etc`. Am I missing something?

Reply:
They are not.

Reply:
So the system-wide hook dir would be under `/etc/xdg/eclean-kernel/hook.d` (unless `XDG_CONFIG_DIRS` is specified). Just to double check: that is what you want?

Reply:
That's what the spec says.

Reply:
Done